### PR TITLE
Add copywrites1

### PR DESCRIPTION
**Pull request by dccowan**
Added copywrite thank you for case histories in which permission was given.

### CODE REVIEW

File: content/case_histories/albany/index.rst [hunk 1]
```diff
@@ -11,9 +11,11 @@ Albany
...
+    This case history follows the role of magnetic and time-domain electromagnetic surveying in the discovery and characterization of the Albany graphite complex, Ontario, Canada. The initial VTEM survey identifying the anomaly was flown by `Geotech Ltd. <http://www.geotech.com>`__ in 2010. Based on the discovery of the deposit by `Zenyatta Ventures Ltd. <http://www.zenyatta.ca>`__ in 2011, `Crone Geophysics Ltd. <http://www.cronegeophysics.com>`__ was contracted to perform supplementary ground-based TDEM surveys in 2013.
 
+**Special Thanks**
+
+Thanks to :ref:`jlegault` and `Geotech Ltd. <http://www.geotech.com>`__ for permission to reproduce figures and adapt text from the source material. This case history was adapted from the expanded abstract `"Airborne and Ground Time-Domain EM results from the Albany Graphite Discovery" <http://www.cronegeophysics.com/wp/wp-content/uploads/2015/04/Albany-Graphite-Air-Ground-TEM_Expanded_abstract_final-13-Feb-2015.pdf>`__.
```
Comment (lheagy) [inline content/case_histories/albany/index.rst]: This is an SEG abstract, so we can Thank them and include the original doi (can you try hunting it down on the SEG website?)

File: content/case_histories/kasted/index.rst [hunk 2]
```diff
@@ -20,8 +20,11 @@ Kasted
 
 .. topic :: Prelude
 
-    This case study is based upon the articles "A regional scale hydrostratigraphy generated from geophysical data of varying age, type and quality by Vilhelmsen (:cite:`Vilhelmsen2016`) and "3D geological modelling of a complex buried-valley network delineated from borehole and AEM data" by Høyer (:cite:`Hoyer2015`).
+    This case history follows the use of the SkyTEM airborne system to delineate subterranean water-bearing structures and assist in groundwater management in Kasted, Denmark.
 
+**Special Thanks**
+
+Thanks to SkyTEM and the `Society of Exploration Geophysicists <http://seg.org/>`__ for permission to reproduce figures and adapt text from the source material. This case study was adapted from the articles "A regional scale hydrostratigraphy generated from geophysical data of varying age, type and quality by Vilhelmsen (:cite:`Vilhelmsen2016`) and "3D geological modelling of a complex buried-valley network delineated from borehole and AEM data" by Høyer (:cite:`Hoyer2015`).
```
Comment (lheagy) [inline content/case_histories/kasted/index.rst]: can you try to hunt down and like the titles to the original doi's from the seg site here?

File: content/case_histories/westplains/index.rst [hunk 3]
```diff
@@ -13,8 +13,11 @@ West Plains
 
 .. topic :: Prelude
 
-    This case history describes the inversion of VTEM and RESOLVE airborne datasets to characterize an orogenic gold deposit. The material found here has been adapted primarily from the paper `"3D parametric hybrid inversion of time-domain airborne electromagnetic data" <https://gif.eos.ubc.ca/sites/default/files/McMillan_parametric.pdf>`__ (:cite:`McMillan2014`). 
+    This case history describes the inversion of VTEM and RESOLVE airborne datasets to characterize an orogenic gold deposit within the Committee Bay Greenstone Belt, Nunavut, Canada. VTEM data were collected in 2003 while RESOLVE data were collected in 2005.
 
+**Special Thanks**
+
+Thanks to the `Society of Exploration Geophysicists <http://seg.org/>`__ for permission to reproduce figures and adapt text from the source material. The material found here has been adapted primarily from the paper `"3D parametric hybrid inversion of time-domain airborne electromagnetic data" <https://gif.eos.ubc.ca/sites/default/files/McMillan_parametric.pdf>`__ (:cite:`McMillan2014`).
```
Comment (lheagy) [inline content/case_histories/westplains/index.rst]: here, we should link to the original doi. we can link to the gif site through the bibliography

File: content/case_histories/westplains/index.rst [hunk 4]
```diff
@@ -13,8 +13,11 @@ West Plains
 
 .. topic :: Prelude
 
-    This case history describes the inversion of VTEM and RESOLVE airborne datasets to characterize an orogenic gold deposit. The material found here has been adapted primarily from the paper `"3D parametric hybrid inversion of time-domain airborne electromagnetic data" <https://gif.eos.ubc.ca/sites/default/files/McMillan_parametric.pdf>`__ (:cite:`McMillan2014`). 
+    This case history describes the inversion of VTEM and RESOLVE airborne datasets to characterize an orogenic gold deposit within the Committee Bay Greenstone Belt, Nunavut, Canada. VTEM data were collected in 2003 while RESOLVE data were collected in 2005.
 
+**Special Thanks**
+
+Thanks to the `Society of Exploration Geophysicists <http://www.seg.org/>`__ for permission to reproduce figures and adapt text from the source material. The material found here has been adapted primarily from the paper `"3D parametric hybrid inversion of time-domain airborne electromagnetic data" <https://gif.eos.ubc.ca/sites/default/files/McMillan_parametric.pdf>`__ (:cite:`McMillan2014`).
```
Comment (lheagy) [inline content/case_histories/westplains/index.rst]: Here, the original doi should be provided (we can find another location to put the link for the paper from the GIF site)

**Comment by dccowan** [comment]
More copywrite permissions still needed.

**Pull request merged**